java2typescript-jackson/src/main/java/java2typescript/jackson/module/writer/WriterPreferences.java

```diff
@@ -23,7 +23,16 @@ public void useStringLiteralTypeForEnums() {
 		this.enumAsStringLiteralType = true;
 	}
 
-	private boolean enumAsStringLiteralType;
+	public void createConstantsForStringLiteralTypeEnums() {
+		this.constantsForStringLiteralTypeEnums = true;
+	}
+
+	public boolean isConstantsForStringLiteralTypeEnums() {
+		return this.constantsForStringLiteralTypeEnums;
+	}
+
+	private boolean enumAsStringLiteralType = false;
+	private boolean constantsForStringLiteralTypeEnums = false;
```

Comment:
Could You place the fields to the beginning of the class? Looks like You followed my bad lead regarding the sort field, that i forgot to move to the beginning of the class :/